### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by ram987815** [community/community#29352] [Actions]
i didnt found canva in studend developer pack

**Comment by onurravli** [comment]
Hi and welcome. Canva is not in the list of GitHub Student Pack Offers anymore. You can find out about other offers <a href="https://education.github.com/pack?sort=popularity&tag=All">here</a>. If it helps, don't forget to mark as answer. Happy coding!

**Comment by ram987815** [comment]
But where I get that please help free canva pro

**Reply by onurravli** [reply]
You better look [this](https://www.canva.com/education/).
